### DISCUSSION TITLE
Copy/Paste between Builders

### DISCUSSION THREAD

**Discussion by dan-dazlab** [vueform/builder#87] [Ideas]
Select a batch of elements and copy them to another Builder.
We use many forms, and we need to refactor them by creating a simpler version of the previous form.
But it's hard to create all elements again on a new form, it would be good to work with selected batches for copy/delete.

**Comment by adamberecz** [comment]
In the meantime you can enable `import: true` in `builder.config.js` that will allow to directly access the builder object, including the schema, which allows for copy/pasting.

**Comment by dan-dazlab** [comment]
Thanks but this is not the same as select specific elements form one builder and past them to another.
It shows all the schema with styles also.
